packages/web3-shared/evm/src/libs/PayloadEditor.ts

```diff
@@ -88,6 +88,7 @@ export class PayloadEditor {
 
     private getRawConfig() {
         const { method, params } = this.payload
+        const coder = ABICoder as unknown as ABICoder.AbiCoder
```

Comment:
If you prefer using plain function calls you could re-export abi coder methods here: `/packages/web3-shared/evm/src/helpers/`